metrics/influx/influx.go

```diff
@@ -0,0 +1,223 @@
+package metrics
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/prebid/prebid-cache/config"
+	"github.com/rcrowley/go-metrics"
+	"github.com/vrischmann/go-metrics-influxdb"
+)
+
+/**************************************************
+ * Constants and global variables
+ **************************************************/
+var TenSeconds time.Duration = time.Second * 10
```

Comment:
Is there a reason for these to be variables rather than constants?

Reply:
No, lol. Changed them to constants